mautic/mautic
# Real-time campaign events triggering

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | y
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:

- [x] add migrations

This PR allow trigger events immediately after contact is added to campaign.

![image](https://user-images.githubusercontent.com/462477/52228606-0d5d8300-28b3-11e9-9510-dc8cf25f8347.png)

#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com/7213). 
If use own test enviroment `run php app/console doctrine:migrations:migrate`
2. Create campaign with some events (actions/conditions) and enable option `Trigger event's realtime`
3. Add contact to campaign
4. See If events are triggered immediately after contact was added to campaign
5. Then disable option. Add another contact to campaigna and see If events are not triggered immediately. I would by wait for cron
